miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -629,3 +632,44 @@ def rotate(self, rotatation: int) -> None:
                 },
             ],
         )
+
+    @command(
+        click.argument("url", type=str),
+        click.argument("md5sum", type=str, required=False),
+        click.argument("size", type=int, default=0),
+    )
+    def set_voice(self, url: str, md5sum: str, size: int):
+        """Upload voice package
+
+        :param str url: URL or path to languaeg pack
+
+        :param str md5sum: MD5 hash for file if URL used
+
+        :param int size: File size in bytes if URL used"""
+        local_url = None
+        server = None
+        if url.startswith("http"):
+            if md5sum is None or size == 0:
+                click.echo(
+                    "You need to pass md5 and file size when using URL for updating.")
+                return
+            local_url = url
+        else:
+            server = OneShotServer(file=url)
+            local_url = server.url()
+            md5sum = server.md5
+            size = len(server.payload)
+
+            t = threading.Thread(target=server.serve_once)
+            t.start()
+            click.echo(f"Hosting file at {local_url}")
+
+        params = [
+            {'piid': 3, 'value': 'MA'},
+            {'piid': 4, 'value': local_url},
+            {'piid': 5, 'value': md5sum},
+            {'piid': 6, 'value': size},
+        ]
+        result_status = self.call_action('set_voice', params=params)
+        if result_status['code'] == 0:
+            click.echo("Installation complete!")
```

Comment:
Let's return the device response just in case someone at downstream wants to use this programatically.
```suggestion
        result_status = self.call_action('set_voice', params=params)
        if result_status['code'] == 0:
            click.echo("Installation complete!")
            
        return result_status
```